CHANGELOG.md

```diff
@@ -76,6 +76,8 @@ To release a new version, please update the changelog as followed:
 - CI Tool:
   - Danger CI has been added to enforce the update of the changelog (by @lgarithm and @DEKHTIARJonathan in #563)
   - https://github.com/apps/stale/ added to clean stale issues (by @DEKHTIARJonathan in #573)
+- API:
+  - Add new layer ElementwiseLambdaLayer (by @One-sixth in #576)
```

Comment:
Incorrect PR number => #579 

Please avoid using 'add of blablabla'. We already know you added something thanks to the Changelog Section.

**Make smthg like this:**
- `ElementwiseLambdaLayer` to perform ????????????????? (by @One-sixth in #579)